lib/timezone/timezone.ex

```diff
@@ -224,6 +224,8 @@ defmodule Timex.Timezone do
       when sign in [?+, ?-],
       do: "Etc/UTC" <> <<sign::utf8, hh::binary, ?:, mm::binary>>
 
+  def name_of(<<tz::binary-size(0)>>), do: {:error, :time_zone_not_found}
```

Comment:
You can change this to be simply `def name_of(""), do: {:error, :time_zone_not_found}`